apps/src/templates/CountryAutocompleteDropdown.jsx

```diff
@@ -75,6 +75,7 @@ export default class CountryAutocompleteDropdown extends Component {
             placeholder={i18n.searchForCountry()}
             labelKey="value"
             matchPos="start"
+            maxHeight={160}
```

Comment:
When you update this PR from "draft" to "ready for review" can you add a description of what issue this fixes?

Ideally we'd let the dropdown expand to fill the screen. Can we clear `overflow: auto;` on the modal div, or does that cause other problems?

Reply:
This PR is a duplicate PR.  see PR: https://github.com/code-dot-org/code-dot-org/pull/30050

I didn't try clearing the overflow, I will test it out.